### ISSUE TITLE
When I create multiple dnn hidden layers using a list, it will only produce 1 dnn layer

### ISSUE THREAD

**Issue by elysego-kumu** [shenweichen/DeepCTR#356]
**Describe the bug(问题描述)**
the parameter DNN hidden units does not add multiple dnn layers when passing a list or a tuple.

**To Reproduce(复现步骤)**
1. model = WDL(linear_feature_columns, dnn_feature_columns, dnn_hidden_units=[128, 64], seed=60,
            dnn_activation="relu", task='binary', dnn_dropout=.35)
model.compile("adam", "binary_crossentropy", metrics=['binary_crossentropy'], )
model.summary()

2. when checking the results of the model summary, there will be only one dnn layer based on the final value in the list
Model: "model"
__________________________________________________________________________________________________
Layer (type)                    Output Shape         Param #     Connected to                     
==================================================================================================
input_1 (InputLayer)             [(None, 1)]          0                                            
__________________________________________________________________________________________________
input_2 (InputLayer)           [(None, 1)]          0                                            
__________________________________________________________________________________________________
input_3 (InputLayer)        [(None, 1)]          0                                            
__________________________________________________________________________________________________
hash_6 (Hash)                   (None, 1)            0           input_1[0][0]                     
__________________________________________________________________________________________________
hash_7 (Hash)                   (None, 1)            0           input_2[0][0]                   
__________________________________________________________________________________________________
hash_8 (Hash)                   (None, 1)            0           input_3[0][0]                
__________________________________________________________________________________________________
sparse_emb_viewer (Embedding)   (None, 1, 8)         31157704    hash_6[0][0]                     
__________________________________________________________________________________________________
sparse_emb_streamer (Embedding) (None, 1, 8)         4969760     hash_7[0][0]                     
__________________________________________________________________________________________________
sparse_emb_day_of_week (Embeddi (None, 1, 8)         64          hash_8[0][0]                     
__________________________________________________________________________________________________
input_4 (InputLayer)              [(None, 1)]          0                                            
__________________________________________________________________________________________________
input_5 (InputLayer)               [(None, 1)]          0                                            
__________________________________________________________________________________________________
no_mask_3 (NoMask)              (None, 1, 8)         0           sparse_emb_viewer[0][0]          
                                                                 sparse_emb_streamer[0][0]        
                                                                 sparse_emb_day_of_week[0][0]     
__________________________________________________________________________________________________
no_mask_4 (NoMask)              (None, 1)            0           month[0][0]                      
                                                                 hour[0][0]                       
__________________________________________________________________________________________________
concatenate_2 (Concatenate)     (None, 1, 24)        0           no_mask_3[0][0]                  
                                                                 no_mask_3[1][0]                  
                                                                 no_mask_3[2][0]                  
__________________________________________________________________________________________________
concatenate_3 (Concatenate)     (None, 2)            0           no_mask_4[0][0]                  
                                                                 no_mask_4[1][0]                  
__________________________________________________________________________________________________
hash (Hash)                     (None, 1)            0           input_1[0][0]                     
__________________________________________________________________________________________________
hash_1 (Hash)                   (None, 1)            0           input_2[0][0]                   
__________________________________________________________________________________________________
hash_2 (Hash)                   (None, 1)            0           input_3[0][0]                
__________________________________________________________________________________________________
flatten (Flatten)               (None, 24)           0           concatenate_2[0][0]              
__________________________________________________________________________________________________
flatten_1 (Flatten)             (None, 2)            0           concatenate_3[0][0]              
__________________________________________________________________________________________________
linear0sparse_emb_viewer (Embed (None, 1, 1)         3894713     hash[0][0]                       
__________________________________________________________________________________________________
linear0sparse_emb_streamer (Emb (None, 1, 1)         621220      hash_1[0][0]                     
__________________________________________________________________________________________________
linear0sparse_emb_day_of_week ( (None, 1, 1)         8           hash_2[0][0]                     
__________________________________________________________________________________________________
no_mask_5 (NoMask)              multiple             0           flatten[0][0]                    
                                                                 flatten_1[0][0]                  
__________________________________________________________________________________________________
no_mask (NoMask)                (None, 1, 1)         0           linear0sparse_emb_viewer[0][0]   
                                                                 linear0sparse_emb_streamer[0][0] 
                                                                 linear0sparse_emb_day_of_week[0][
__________________________________________________________________________________________________
no_mask_1 (NoMask)              (None, 1)            0           input_4[0][0]                      
                                                                 input_5[0][0]                       
__________________________________________________________________________________________________
concatenate_4 (Concatenate)     (None, 26)           0           no_mask_5[0][0]                  
                                                                 no_mask_5[1][0]                  
__________________________________________________________________________________________________
concatenate (Concatenate)       (None, 1, 3)         0           no_mask[0][0]                    
                                                                 no_mask[1][0]                    
                                                                 no_mask[2][0]                    
__________________________________________________________________________________________________
concatenate_1 (Concatenate)     (None, 2)            0           no_mask_1[0][0]                  
                                                                 no_mask_1[1][0]                  
__________________________________________________________________________________________________
dnn (DNN)                       (None, 64)           11712       concatenate_4[0][0]              
__________________________________________________________________________________________________
linear (Linear)                 (None, 1)            2           concatenate[0][0]                
                                                                 concatenate_1[0][0]              
__________________________________________________________________________________________________
dense (Dense)                   (None, 1)            64          dnn[0][0]                        
__________________________________________________________________________________________________
no_mask_2 (NoMask)              (None, 1)            0           linear[0][0]                     
__________________________________________________________________________________________________
add (Add)                       (None, 1)            0           dense[0][0]                      
                                                                 no_mask_2[0][0]                  
__________________________________________________________________________________________________
prediction_layer (PredictionLay (None, 1)            1           add[0][0]                        
==================================================================================================
Total params: 40,655,248
Trainable params: 40,655,248
Non-trainable params: 0

**Operating environment(运行环境):**
 - python-3.6
 - tensorflow-2.4.1
 - deepctr-0.8.5

**Comment by zanshuxun** [comment]
the multiple dnn layers are wrapped  in the DNN class

**Comment by elysego-kumu** [comment]
Got it. Thanks!
